jcl/src/java.base/share/classes/openj9/tools/attach/diagnostics/base/DiagnosticUtils.java

```diff
@@ -126,6 +127,7 @@ public static String makeJcmdCommand(String[] options, int skip) {
 	}
 
 	private static native String getHeapClassStatisticsImpl();
+	private static native String triggerDumpsImpl(String dumpOptions, String event) throws InvalidDumpOptionExceptionBase;
```

Comment:
Where will this be used?

Reply:
This will be used by `jcmd`'s `Dump.*` commands.